lib/rouge/lexers/sqf.rb

```diff
@@ -0,0 +1,602 @@
+          triggertype true try turretlocal turretowner turretunit tvadd tvclear
+          tvcollapse tvcollapseall tvcount tvcursel tvdata tvdelete tvexpand
+          tvexpandall tvpicture tvpictureright tvsetcolor tvsetcursel tvsetdata
+          tvsetpicture tvsetpicturecolor tvsetpicturecolordisabled
+          tvsetpicturecolorselected tvsetpictureright tvsetpicturerightcolor
+          tvsetpicturerightcolordisabled tvsetpicturerightcolorselected
+          tvsetselectcolor tvsettext tvsettooltip tvsetvalue tvsort
+          tvsortbyvalue tvtext tvtooltip tvvalue type typename typeof uavcontrol
+          uinamespace uisleep unassigncurator unassignitem unassignteam
+          unassignvehicle underwater uniform uniformcontainer uniformitems
+          uniformmagazines unitaddons unitaimposition unitaimpositionvisual
+          unitbackpack unitisuav unitpos unitready unitrecoilcoefficient units
+          unitsbelowheight unlinkitem unlockachievement unregistertask
+          updatedrawicon updatemenuitem updateobjecttree
+          useaiopermapobstructiontest useaisteeringcomponent
+          useaudiotimeformoves userinputdisabled vectoradd vectorcos
+          vectorcrossproduct vectordiff vectordir vectordirvisual vectordistance
+          vectordistancesqr vectordotproduct vectorfromto vectormagnitude
+          vectormagnitudesqr vectormodeltoworld vectormodeltoworldvisual
+          vectormultiply vectornormalized vectorup vectorupvisual
+          vectorworldtomodel vectorworldtomodelvisual vehicle
+          vehiclecargoenabled vehiclechat vehicleradio
+          vehiclereceiveremotetargets vehiclereportownposition
+          vehiclereportremotetargets vehicles vehiclevarname velocity
+          velocitymodelspace verifysignature vest vestcontainer vestitems
+          vestmagazines viewdistance visiblecompass visiblegps visiblemap
+          visibleposition visiblepositionasl visiblescoretable visiblewatch
+          waituntil waves waypointattachedobject waypointattachedvehicle
+          waypointattachobject waypointattachvehicle waypointbehaviour
+          waypointcombatmode waypointcompletionradius waypointdescription
+          waypointforcebehaviour waypointformation waypointhouseposition
+          waypointloiterradius waypointloitertype waypointname waypointposition
+          waypoints waypointscript waypointsenableduav waypointshow
+          waypointspeed waypointstatements waypointtimeout
+          waypointtimeoutcurrent waypointtype waypointvisible weaponaccessories
+          weaponaccessoriescargo weaponcargo weapondirection weaponinertia
+          weaponlowered weapons weaponsitems weaponsitemscargo weaponstate
+          weaponsturret weightrtd west wfsidetext while wind winddir windrtd
+          windstr wingsforcesrtd with worldname worldsize worldtomodel
+          worldtomodelvisual worldtoscreen
+        )
+      end
+
+      # high priority for filename matches
+      def self.analyze_text(*)
```

Comment:
analyze_text is no longer availabe on the maste branch.

See https://github.com/jneen/rouge/pull/763 for details